std/random.d

```diff
@@ -2634,6 +2634,104 @@ do
     assert(i == 0);
 }
 
+/+ @nogc bool array designed for RandomCover.
+- constructed with an invariable length
+- small length means 0 alloc and bit field (if up to 32 and 64 elements to cover)
+- bigger length means non-GC heap allocation(s) and dealloc. +/
+private struct RandomCoverChoices
+{
+    private static immutable emsg = "No memory available to store random cover choices";
+    private void* buffer;
+    private immutable size_t _length;
+    private immutable bool hasPackedBits;
+
+    this(this) nothrow @nogc @trusted
+    {
+        import core.stdc.stdlib : malloc;
+        import core.stdc.string : memmove;
+
+        if (!hasPackedBits && buffer !is null)
+        {
+            void* nbuffer = malloc(_length);
```

Comment:
this branch is not covered anymore since the switch to the bit field. There should be a test with 65 or more elements to cover...